### DISCUSSION TITLE
How to build to exact source kernel version (source code), without booting?

### DISCUSSION THREAD

**Discussion by L2NE-dev** [NVIDIA/open-gpu-kernel-modules#552] [Q&A]
I recently built linux kernel, but I don't want to boot for install drivers, I want to pre-build drivers into that kernel. How to do that?

**Comment by aritger** [comment]
You should be able to point the open-gpu-kernel-modules build at a different target kernel with the `SYSSRC` variable.  E.g., `make modules -j32 SYSSRC=/path/to/new/kernel`
